### PR TITLE
fix(compliance): updated the npm packages

### PR DESCRIPTION
**Pull request by RishikeshDarandale**
Updated the outdated npm packages. All the tests are running with success.

Updated the outdated npm packages.
    
Only following dev dependencies are pending to update
    
```console
Package          Current  Wanted  Latest  Location
readable-stream    2.0.6   2.0.6   3.1.1  soap
sinon             1.17.7  1.17.7   7.2.4  soap
```

### CODE REVIEW

**Comment by RishikeshDarandale** [comment]
@jsdevel `coveralls` is [failed](https://travis-ci.org/vpulim/node-soap/jobs/496363378) on for `node7` due to timeout, I think it will be increased as per #1047.

Lets enable [dependabot](https://dependabot.com/) for dependency management.

**Comment by RishikeshDarandale** [comment]
@jsdevel , I have rebased this PR, but looks like I need to write more test cases of uncovered lines to pass 94%. Though I have not modified any code here!  :crossed_fingers:

File: test/wsdl-test.js [hunk 1]
```diff
@@ -6,232 +6,229 @@ var fs = require('fs'),
...
-      for (i = 0; i < spy.callCount; i++) {
-        spyCall = spy.getCall(i);
-        if (spyCall.args[0]) {
-          assert.notEqual(spyCall.args[0].$type, "RabbitHole");
-        }
-      }
+describe('WSDL Parser (non-strict)', () => {
+  fs.readdirSync(__dirname+'/wsdl').forEach(function(file) {
+    if (!/.wsdl$/.exec(file)) return;
+    it('should parse and describe '+file, (done) => {
```
Comment (jsdevel) [inline test/wsdl-test.js]: i'm somewhat leary of this.  seems the test has been rewritten.  is there a way to revert this?

Comment (RishikeshDarandale) [reply test/wsdl-test.js]: I have not rewritten any test cases from this file, they are arranged under two suites only. the only change is from `function () {}` to `() => {}`. If you search the same, you should be able to see it.

Comment (jsdevel) [reply test/wsdl-test.js]: hmm, ok, well see if you can get coverage back up

Comment (RishikeshDarandale) [reply test/wsdl-test.js]: @jsdevel , this is the max I can get the coverage by adding the new tests.  I almost did one on one file conversion for missing lines with previous commit, but looks to be same as the previous. The coverage gets dropped with upgrade of `nyc`. 

with `nyc@11.4.1`, coverage becomes `94.32` and with `nyc@13.3.0`, it becomes `93.22`. The only thing with older verison is security vulnerability.

```console
$ npm install nyc@11.4.1
+ nyc@11.4.1
added 96 packages from 87 contributors, removed 47 packages, updated 70 packages and audited 2272 packages in 58.627s
found 23 vulnerabilities (13 low, 9 moderate, 1 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

If it's not worth merging, then lets close this.

Thanks!

**Comment by jsdevel** [comment]
thanks @RishikeshDarandale !

**Pull request merged**